test/Makefile

```diff
@@ -38,6 +38,7 @@ endif
 EVENTO        = Event.$(ObjSuf) EventDict.$(ObjSuf)
 EVENTS        = Event.$(SrcSuf) EventDict.$(SrcSuf)
 EVENTSO       = libEvent.$(DllSuf)
+EVENTLIBS     = -lTreePlayer
```

Comment:
Yes but -l is not platform-neutral :) [So the -l indeed need to be avoided on windows.]